### DISCUSSION TITLE
Marketing Campaigns icon additions

### DISCUSSION THREAD

**Discussion by kyliepedersen** [twilio-labs/paste#562] [Request: Icon] [answered]
Hello! Jason Walker used to create additional icons the Marketing Campaigns team needed for the app as well as upload them to the [SendGrid styleguide](http://styleguide.sendgrid.com/icons.html). There hasn't been any updates on whether MC will be transitioning from using UI-Components to Paste in the near future and in the meantime, we need some new icons for some designs we are trying to complete. Is there someone who took over Jason's responsibilities that could help me out? Specifically, we would like to add the icon from the screenshot below, in addition to a "power" icon such as below as well. If someone could create the svg file, I will be able to upload it and use it in our FE app. Thanks in advance for the help!!
<img width="235" alt="Screen Shot 2020-07-08 at 10 31 05 AM" src="https://user-images.githubusercontent.com/10856504/86945397-31f00700-c106-11ea-9f89-f68bb4566c31.png">
![image (3)](https://user-images.githubusercontent.com/10856504/86945590-71b6ee80-c106-11ea-88c5-8eeb92da4684.png)

**Comment by TheSisb** [comment]
Hi @kyliedale, interesting question with multiple possible answers.  I have some questions to gain additional context on your problem:

1. Do you just need the SVG? 
2. Do you have Streamline access to download these icons or would you need us to grab them?
3. Can you share how you plan to use the Power icon? I'm curious about the use-case to see if it would "fit" in Paste, which may be a path forward.
4. Are you going to use the SVG directly in your FE app or are you planning on adding it to an icon font? (Alternatively, you can [use the tooling we use](https://github.com/twilio-labs/svg-to-react) for Paste icons and add it to your project directly. [Why we do it this way](https://github.com/twilio-labs/svg-to-react/blob/master/RATIONALE.md))

**Comment by kyliepedersen** [comment]
Hi @TheSisb, thanks for your reply! 

1. I just need the SVG so that we can add it to our [component library](https://github.com/sendgrid/ui-components). 
2. I don't believe I have Streamline access to download the icons.
3. It's for the `Set Live` option for Automation email campaigns. (I can give more context on that if you need it!)
4. I'm planning on adding it to our component library, not using it directly in our app. [These instructions](https://github.com/sendgrid/ui-components#updating-icons) might be out of date, but might help to give more context.

**Comment by richbachman** [comment, marked as answer]
@kyliedale just adding a note after our chat the other day. This may be a one off task, so in order to move it forward the options are:

1. Choose another icon that already exists in StyleGuide.
2. Use [svg-to-react](https://github.com/twilio-labs/svg-to-react) to create a new icon for use in ui-components.

If MC is still going to be actively worked on, we may need to renew the IcoMoon license so we can update the icon font. Kylie will know more in a couple weeks, and will let us know by adding a new discussion thread.
